Documentation/integrations.md

```diff
@@ -2,7 +2,7 @@
 This document tracks projects that integrate with rkt. [Join the community](https://github.com/rkt/rkt/), and help us keep the list up-to-date.
 
 ## Projects 
-[Kubernetes](https://kubernetes.io/docs/getting-started-guides/rkt/): a enterprise grade container orchestration solution. Kubernetes offers rkt support as an alternative to the standard install.
+[Kubernetes](https://github.com/kubernetes-incubator/rktlet/blob/master/docs/getting-started-guide.md): an enterprise grade container orchestration solution. Kubernetes offers rkt support as an alternative to the standard install.
```

Comment:
why not linking to https://github.com/kubernetes-incubator/rktlet?

Reply:
yeah, why not?